src/service/viewer-impl.js

```diff
@@ -68,19 +68,34 @@ export const ViewportType = {
 /**
  * Visibility state of the AMP document.
  * @enum {string}
- * @private
  */
 export const VisibilityState = {
-
   /**
-   * Viewer has shown the AMP document.
+   * The AMP document is currently active and visible.
    */
   VISIBLE: 'visible',
 
   /**
-   * Viewer has indicated that AMP document is hidden.
+   * The AMP document is active but the browser tab is not.
+   */
+  HIDDEN: 'hidden',
+
+  /**
+   * The AMP document is visible but it should stop active playback because
```

Comment:
"The AMP document is visible, but the user has starting switching away from the it. The runtime may stop active playback". Consider using "swiping" instead of "switching".